addon/components/frost-date-picker.js

```diff
@@ -68,7 +94,8 @@ export default Component.extend(EventsProxyMixin, {
     run.scheduleOnce('sync', this, function () {
       const pikadayOptions = {
         field: this.$('input')[0],
-        onSelect: run.bind(this, this._onSelect)
+        onSelect: run.bind(this, this._onSelect),
```

Comment:
oh, just realized was looking at time picker not the date picker, will take a look at the date picker :)